ogr/abstract.py

```diff
@@ -577,6 +577,15 @@ def get_pr_comments(
         """
         raise NotImplementedError()
 
+    def get_pr_commits(self, pr_id: int) -> List[str]:
+        """
+        Get list of pull-request commits.
+
+        :param pr_id: int
+        :return: [str]
```

Comment:
I guess the `str` means the commit hash, but you might make that explicit. Thanks.